definitions/npm/countries-and-timezones_v1.x.x/test_countries-and-timezones_v1.x.x.js

```diff
@@ -0,0 +1,21 @@
+// @flow
```

Comment:
Place the test file along side the flow-specific implementation. Tests may vary across flow versions, so we keep them versioned.